.rustfmt.toml

```diff
@@ -0,0 +1 @@
+max_width = 80
```

Comment:
I know this may not be popular, and I'm fine to remove it.

But, I always code in 80 character width terminals... and it's very annoying having wrapping lines...

Reply:
As long as the formatter does it, I don't care. =)